.github/workflows/deploy-backoffice.yml

```
@@ -0,0 +1,79 @@
+        required: true
+        default: 'dev'
+
+jobs:
+  build:
+    name: Build Backoffice App
+    runs-on: ubuntu-latest
+    environment: ${{ github.event_name == 'push' && github.ref_name || inputs.environment }}
+    steps:
+      # Checkout the repository
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)